src/openrct2/localisation/string_ids.h

```diff
@@ -3665,6 +3665,9 @@ enum {
 	STR_UNIT1DP_SUFFIX_FEET = 6011,
 	STR_UNIT1DP_NO_SUFFIX = 6012,
 
+	STR_RIDE_INCOME_ADMISSION_PAY_FOR_ENTRY_TIP = 6013,
+	STR_ADMISSION_PRICE_PAY_PER_RIDE_TIP = 6014,
```

Comment:
**Bug (which makes me require changes):**
It shows the wrong text in the wrong order, see screenshot